docs/en/essentials/named-views.md

```diff
@@ -27,3 +27,58 @@ const router = new VueRouter({
 ```
 
 A working demo of this example can be found [here](https://jsfiddle.net/posva/6du90epg/).
+
+## Nested Named Views
+
+While being quite complex, it is possible to combine named views with nested view. When doing so, you will also need to name nested `router-view` components used. Let's take a Settings panel example:
+
+```
+/settings/emails                                       /settings/profile
++-----------------------------------+                  +------------------------------+
```

Comment:
We should put some image instead.

Reply:
Let's keep this for the moment. An Image would be indeed better